composer.json

```diff
@@ -18,18 +18,18 @@
     ],
     "require": {
         "doctrine/orm": "^2.5.3",
-        "php": "^7.1",
-        "sulu/sulu": "^2.0.3",
-        "symfony/config": "^4.3",
-        "symfony/console": "^4.3",
-        "symfony/dependency-injection": "^4.3",
-        "symfony/event-dispatcher": "^4.3",
+        "php": "^7.2",
+        "sulu/sulu": "^2.0.7 || ^2.1",
```

Comment:
`^2.1` is not needed as included in `^2.0.7` and I would stay with `^2.0.3` we there is no requirement for using a newer patch version